### ISSUE TITLE
How to get email body ?

### ISSUE THREAD

**Issue by ruslanx3m** [XielQs/mail.tm-api#3]
How to get email body ?

```
const MailTM = require('mail.tm-api');


(async () => {

    const account = await MailTM.loginAccount('xxxx', 'yyy');

    console.log(await account.fetch());

    console.log(await account.emails.fetchAll()); //! Error: Cannot read property 'fetchAll' of undefined

})();
```

Error: Cannot read property 'fetchAll' of undefined

do it exists "emails" ?

**Comment by XielQs** [comment]
That should be account.mails, there is a typo sorry! and don't leak your credentials!!

**Comment by XielQs** [comment]
Fixed with [11f4acc](https://github.com/XielQs/mail.tm-api/commit/11f4acc705273b6b04771e2d626c257835b0d343) ❤️
